### ISSUE TITLE
club100.hk - cannot login

### ISSUE THREAD

**Issue by ReporterX** [meh/smart-referer#102]
[https://www.club100.hk/](https://www.club100.hk/)

You cannot login if the addon is on.
It is hard to know what's causing this if you don't have an account to test.
Perhaps someone teaches me how to check so I can fix it myself.
Thank you.

A sub-optimal quick fix is to add it to the exception.
Source: club100.hk
Destination: *

**Comment by alexander255** [comment]
Thanks for reporting this I can indeed not reproduce (create and test an account there) without a Hong-Kong transport pass number + mobile phone number it seems. I haven't tried using a HK phone number from one of the many SMS receive websites yet.

If you want to help out that'd be much appreachiated however! I had already the described the general task of gathering the necessary information [in other issues](https://github.com/meh/smart-referer/issues/69#issuecomment-342484721):
> […], please open the network analyser panel (Ctrl+Shift+E) on that site, perform whatever action is broken (if its the entire page then just reload) and upload a screenshot of the result. It should look similar to this (make the panel as large as possible before taking the screenshot, if the content still doesn't fit into the panel, please take one screenshot per page and post all of them):
![screenshot_20171031_203708](https://user-images.githubusercontent.com/246386/32245194-46c45428-be7b-11e7-840d-758f20774af9.png)

This then gets us a list of host names (forth column) to try. I do basically the same thing when I look for whitelist entries to add.

**Comment by ReporterX** [comment]
Those SMS receive websites aren't good. I tried using some numbers. I kept waiting for several hours but the sms never came. Funny enough I saw a few SMS from those numbers which were received in several hours or one/two days ago.

Something weird happened. I removed the sub-optimal whtielist. I tried to log in without opening the network analyser. It failed the first time. I opened the network analyser and logged in again. It worked suddenly this time.

I cleared the cookies and tried again. I did not open the network analyser. I logged in successfully again. o.O What happened?
